docs/build/building-apps/00-runtime.md

```diff
@@ -0,0 +1,153 @@
+---
+sidebar_position: 1
+---
+
+# What is `runtime`?
+
+The `runtime` package in the Cosmos SDK provides a flexible framework for configuring and managing blockchain applications. It serves as the foundation for creating modular blockchain applications using a declarative configuration approach.
+
+## Overview
+
+The runtime package acts as a wrapper around the `BaseApp` and `ModuleManager`, offering a hybrid approach where applications can be configured both declaratively through configuration files and programmatically through traditional methods.
+It is a layer of abstraction between `baseapp` and the application modules that simplifies the process of building a Cosmos SDK application.
+
+## Core Components
+
+### App Structure
+
+The runtime App struct contains several key components:
+
+```go
+type App struct {
+    *baseapp.BaseApp
+    ModuleManager      *module.Manager
+    UnorderedTxManager *unorderedtx.Manager
```

Comment:
We don't have this anymore right @technicallyty

Reply:
removed.